portfolio/src/main/webapp/script.js

```diff
@@ -42,14 +42,14 @@ async function getGreetingUsingAsyncAwait() {
 function loadComments() {
   fetch('/data').then(response => response.json()).then((comments) => {
     const commentListElement = document.getElementById('comments-container');
-    comments.forEach((comments) => {
-      commentListElement.appendChild(createCommentElement(comments));
+    comments.forEach((comment) => {
+      commentListElement.appendChild(createCommentElement(comment.name, comment.message, comment.time));
     });
   });
 }
 
-function createCommentElement(comment) {
+function createCommentElement(name, message, time) {
```

Comment:
I should have caught this earlier, but this function needs a JsDoc comment. See https://google.github.io/styleguide/jsguide.html#jsdoc for details, but you will want to at least add docs for your param names and their types, and the return type.

Reply:
Like this, or am I missing something?
/**
 * @param {string} name The name of the user who commented.
 * @param {string} message The message body of a comment post.
 * @param {string} time The time of a comment post.
 */